pkg/types/apisix/v1/types.go

```diff
@@ -81,6 +81,13 @@ const (
 	DefaultUpstreamTimeout = 60
 )
 
+var IngressSchemeMap map[string]struct{} = map[string]struct{}{
```

Comment:
The name doesn't quite fit because these valid schemes are not just for Ingress. `ValidSchemes` would be better, but I can approve this.

Reply:
Yes, I also agree with naming `ValidSchemes`  then we can merge  it